js/functions.js

```diff
@@ -79,6 +79,12 @@
 		// Toggle `focus` class to allow submenu access on tablets.
 		function toggleFocusClassTouchScreen() {
 			if ( window.innerWidth > 910 ) {
+				$( 'body' ).on( 'touchstart.twentysixteen', function( ) {
+					if ( ! $( event.target ).closest( '.main-navigation li' ).length )
+					{
```

Comment:
Before testing this further, can you move this curly brace to the same line as the `if` check for the sake of consistency?